src/test/java/com/exasol/adapter/jdbc/ColumnMetadataReaderTest.java

```diff
@@ -221,29 +215,23 @@ void testBigInteger() throws SQLException {
 
     @Test
     void testBigIntegerWithPrecision() throws SQLException {
-        final int expectedPrecision = 35;
-        assertSqlTypeWithPrecisionConvertedToExasolType(Types.BIGINT, expectedPrecision,
-                DataType.createDecimal(expectedPrecision, 0));
+        final int precision = 17;
+        assertSqlTypeWithPrecisionConvertedToExasolType(Types.BIGINT, precision, 0,
```

Comment:
Why did you reduce the precision here so much? I think the original idea was to test the limit. Maybe parameterizing this test would be a good idea?